lib/std/rand/ziggurat.zig

```diff
@@ -16,7 +16,7 @@ pub fn next_f64(random: *Random, comptime tables: ZigTable) f64 {
     while (true) {
         // We manually construct a float from parts as we can avoid an extra random lookup here by
         // using the unused exponent for the lookup table entry.
-        const bits = random.scalar(u64);
+        const bits = random.int(u64);
         const i = @as(usize, bits & 0xff);
```

Comment:
```suggestion
        const i = @as(usize, @truncate(u8, bits));
```

Change this and you can also close #2939